autoPlay.user.js

```diff
@@ -16,7 +16,7 @@
 	"use strict";
 
 // OPTIONS
-var clickRate = 10;
+var clickRate = 20;
```

Comment:
Simply forgot to mention it in File comment but added it to Title, also i not really wanted to fork it, happened by accident, but then decided to fix that Bug and to try that pull Thing. ;)